src/Sylius/Bundle/AddressingBundle/Controller/ProvinceController.php

```diff
@@ -78,4 +90,15 @@ protected function createProvinceChoiceForm(CountryInterface $country)
             'empty_value' => 'sylius.form.province.select',
         ]);
     }
+
+    /**
+     * @return FormInterface
+     */
+    protected function createProvinceTextForm()
+    {
+        return $this->get('form.factory')->createNamed('sylius_address_province', 'text', null, [
```

Comment:
I know it's a very simple form, but I'm just wondering wouldn't it be better to create new form type rather than create it in controller function.